frontend/online-shop/src/components/productsList/DescriptionCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, GridListTileBar } from "@material-ui/core";
+import { useStyle } from "../../styles/productsList/DescriptionCardStyle";
+import { ThemeProvider } from "@material-ui/styles";
+import { standardTheme } from "../../themes/StandardTheme";
+import { Link } from "react-router-dom";
+
+export interface Product {
```

Comment:
Extract the interfaces (models) in a separate package called `models`. One file per model (e.g. `product.tsx`)